### DISCUSSION TITLE
Unable to install on Windows 11 - crash on startup ?

### DISCUSSION THREAD

**Discussion by tim1mw** [valinet/ExplorerPatcher#713] [General]
I'm trying to install this on a clean Windows 11 system, but the installer doesn't seem to do anything. When double clicking I get prompted to allow the installation, after clicking yes, my back drop briefly turns grey and nothing further happens. I gain the impression the installer is crashing on start up.

**Comment by valinet** [comment]
Try opening Task Manager, run the installer, then when the screen becomes gray, check to see whether ep_setup.exe is still running in Task Manager. Then, check if explorer.exe is still running and kill that and see if it restarts successfully with the Windows 10 taskbar after that.

**Reply by peterstavrou** [reply]
I am having the same issue on Windows 11 Pro, version: 21H2, OS build: 22000.376.
The ep_setup.exe app crashes right away after a gray screen, it doesn't even appear in the Task Manager and explorer.exe is not appearing in there either.

**Reply by valinet** [reply]
Try to collect a crash dump for ep_setup. That may tell us where it is crashing and why. Use [these instructions](https://docs.microsoft.com/en-us/windows/win32/wer/collecting-user-mode-dumps) to enable full crash dumps and run ep_setup again.

**Reply by peterstavrou** [reply]
Tried the above instructions and no crash dump gets generated.

However, I did realize after the program crashes it still creates a folder here:
C:\Program Files\ExplorerPatcher

When I go in to run the ep_setup.exe file from there, I get the below message.
I have tried killing explorer.exe with command prompt and then running it but still the same message.
`taskkill /f /im explorer.exe`

Once it crashes again, the ep_setup.exe file disappears from the ExplorerPatcher folder.
```
An error has occurred while servicing this product.
This is most likely caused by one or more of the backup files from the previous update still being in use. Unlocking the files should fix this issue.

Troubleshooting steps:
* Close and reopen the "Properties" dialog if it is currently open.
* Kill and restart all "explorer.exe" processes.
* If you have registered this application as a shell extension, then restarting the computer will probably fix this.
* Lastly, reboot the computer and try again.
```

**Reply by jffortier249** [reply]
Same for me
Just unpdated to Win11 amd I have been searching how to get my tiles back for the past 3 hours !! Luckely this is my home PC, I wanted to test it on my desktop home before lauching an office wide update in the next week... 

So I launch the app as admin, nothing really happen, I see a flash, explorer close and start, I have the folder C:\Program Files\ExplorerPatcher, reboot several time.... nothing ?!?! start the app ep_dwm.exe ?! nothing happen

Win11 pro
21H2
22000.493

**Comment by Jayblesz** [comment]
I am having the same issue. On AMD 5600X Windows 11 pro- white screening as above. 
I have also tried the previous version (.1) that is currently working on my windows 11 Home Intel 12th gen laptop, and it doesn't work either.

I click the installation exe and it white screens, but does in fact seem to install, but when I go to open the setup in C folder it white screens and gives me the message as per OP's error post.

How can I fix this please, and/or how can I give more details to assist in the fix?

**Reply by Jayblesz** [reply]
Edit: I seemed to be able to get it to work miraculously, by instead finding (with a start menu search) the Properties (ExplorerPatcher) app and opening that instead of the file within rhebC drive. I can now configure it.

**Comment by hdbowker** [comment]
I was able to break the cycle of endless windows desktop crashes.
Using the taskmanager GUI, start a new process, navigate to C:/program files/explorerpatcher
Right click on both executables and run compatibility troubleshooter (it should set to Windows 8)
Test the compatibility using Windows 8.  
I had to repeat the process two or three times, but eventually the desktop explorer crashes stop and I was able to run multiple unstacked/ungrouped windows of the same applications.

My suggestion is then: download the executable, but do not Run it.  Change property settings using file explorer by navigating to C:/program files/explorerpatcher  and change both compatibility settings to windows 8 and run as administrator.
Save changes before trying to run.

Windows 11 Home Activated
Fresh Install 10/2/23
7950x3d
64gb ram
2 x 1tb nvme 980 samsung 
GB B650 Aorus Master
Explorerpatcher downloaded from the readme file link using Google Chrome.
